.storybook/manager-head.html

```diff
@@ -158,4 +172,58 @@
     );
     bookmarkHollowIcon.setAttribute("fill-rule", "evenodd");
   });
+
+  window.addEventListener("load", () => {
+    changeElementDimension(
+      document.querySelector('div[role="main"]'),
+      "width",
+      10
+    );
+    changeElementDimension(
+      document.querySelector('div[role="main"]'),
+      "height",
+      20
+    );
+    changeElementDimension(
+      document.querySelector('div[role="main"] > div > div ~ div'),
+      "height",
+      20
+    );
+  });
+
+  window.addEventListener(
```

Comment:
We can use ResizeObserver to offload some js work!

https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver

Reply:
After implementing the feature using ResizeObservers, I came across this comment from the W3C technical architecture group about using Events and Observers appropriately, and I opted for a more traditional method. 🤔
What do you think? Should I change it to an observer?

https://w3ctag.github.io/design-principles/#events-vs-observers

p.s.
I cleaned up the code by making certain adjustments. 🤓

Reply:
Ooh thanks for sharing the link! That doc looks great ⚡Will need to read through the whole thing later too ahaha!

Yeah I think you're right that using resize event here is better, because we're not getting any element dimensions anyway, so no extra js loops saved 😆 Thanks for pointing that out!

Reply:
Thanks for checking that out 🙏🏻🥳